test.js

```diff
@@ -25,11 +25,11 @@ describe('yeoman-assert', () => {
 
     it('reject a file that does not exist', () => {
       assert.throws(yoAssert.file.bind(yoAssert, 'etherealTestFile'));
-    });
+    }, /.*AssertionError.*/);
```

Comment:
Suggestion: I'd revert these. I don't plan on blocking on this, but just stating a preference: I don't think these are worth adding. Nothing prevents the comments from getting out of date. And we already know some error is being thrown. If we really wanted to assert on which error that is, we can always check its class/constructor/name.

Totally up to you though, feel free to ignore this suggestion. 🙂